healthtools_ke_api/__init__.py

```diff
@@ -16,6 +19,9 @@
 app.register_blueprint(doctors_api, url_prefix='/doctors')
 app.register_blueprint(nurses_api, url_prefix='/nurses')
 app.register_blueprint(clinical_officers_api, url_prefix='/clinical-officers')
+app.register_blueprint(nhif_inpatient_api, url_prefix='/nhif-inpatient')
+app.register_blueprint(nhif_outpatientcs_api, url_prefix='/nhif-outpatientcs')
```

Comment:
Let's have the the url as `/nhif-outpatient-cs`.

Reply:
changed to ```url_prefix='/nhif-outpatient-cs'```